src/transformers/__init__.py

```diff
@@ -1631,6 +1631,7 @@
             "ChineseCLIPModel",
             "ChineseCLIPPreTrainedModel",
             "ChineseCLIPTextModel",
+            "ChineseCLIPTextTransformer",
```

Comment:
Should we add `ChineseCLIPTextTransformer`, isn't it an internal module only?
The same question regarding all the following additions of `*Transformer` class to imports and docs, maybe we can make `ChineseCLIPTextTransformer` just an `nn.Module` and avoid adding it to imports.

Reply:
yes, not sold on keeping it - probably safer to remove it, I think I'll do that

Reply:
+1 on keeping it an internal module

Reply:
done!